projects/sandbox/train/tests/test_train.py

```diff
@@ -155,14 +160,58 @@ def test_train(
         return_value=np.random.randn(1, 2, 1024 * sample_rate),
     )
 
-    with background_fnames_mock, waveforms_mock, background_mock as _, _, _:
-        train_dataset, validator, preprocessor = main(
+    glitch_mock = patch(
+        "train.utils.get_glitch_fnames",
+        return_value={
+            ifo: [
+                f"{ifo}-glitches-10000-1024.h5",
+                f"{ifo}-glitches-20024-1024",
```

Comment:
If tests are passing I suppose this doesn't affect anything, but this file name is missing a `.h5`